bitnami/jaeger/values.yaml

```diff
@@ -0,0 +1,1131 @@
+  ##
+  extraVolumes: []
+  ## @param agent.initContainers Add additional init containers to the jaeger pods
+  ##
+  initContainers: []
+  ## @param agent.sidecars Add additional sidecar containers to the jaeger pods
+  ##
+  sidecars: []
+
+
+migration:
+  ## @param migration.podLabels Additional pod labels
+  ## ref: https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/
+  ##
+  podLabels: {}
+  ## @param migration.podAnnotations Additional pod annotations
+  ## ref: https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/
+  ##
+  podAnnotations: {}
+  ## @param migration.annotations Provide any additional annotations which may be required.
+  ## This can be used to set the LoadBalancer service type to internal only.
+  ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#internal-load-balancer
+  ##
+  annotations: {}
+  ## Pod security context
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod
+  ## @param migration.podSecurityContext.enabled Enabled Jaeger pods' Security Context
+  ## @param migration.podSecurityContext.fsGroup Set Jaeger pod's Security Context fsGroup
+  ##
+  podSecurityContext:
+    enabled: true
+    fsGroup: 1001
+  ## @param migration.extraEnvVarsCM Name of existing ConfigMap containing extra env vars
+  ##
+  extraEnvVarsCM: ""
+  ## @param migration.extraEnvVarsSecret Name of existing Secret containing extra env vars
+  ##
+  extraEnvVarsSecret: ""
+  ## @param migration.extraVolumeMounts Optionally specify extra list of additional volumeMounts for jaeger container
+  ##
+  extraVolumeMounts: []
+  ## Jaeger pods' resource requests and limits
+  ## ref: https://kubernetes.io/docs/user-guide/compute-resources/
+  ## Minimum memory for development is 4GB and 2 CPU cores
+  ## Minimum memory for production is 8GB and 4 CPU cores
+  ## ref: http://docs.datastax.com/en/archived/jaeger/2.0/jaeger/architecture/architecturePlanningHardware_c.html
+  ##
+  ## We usually recommend not to specify default resources and to leave this as a conscious
+  ## choice for the user. This also increases chances charts run on environments with little
+  ## resources, such as Minikube. If you do want to specify resources, uncomment the following
+  ## lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  ## @param migration.resources.limits The resources limits for Jaeger containers
+  ## @param migration.resources.requests The requested resources for Jaeger containers
+  ##
+  resources:
+    ## Example:
+    ## limits:
+    ##    cpu: 2
+    ##    memory: 4Gi
+    ##
+    limits: {}
+    ## Examples:
+    ## requests:
+    ##    cpu: 2
+    ##    memory: 4Gi
+    ##
+    requests: {}
+  ## @param migration.extraVolumes Optionally specify extra list of additional volumes for jaeger container
+  ##
+  extraVolumes: []
+
+## @section Cassandra storage pod
+##
+cassandra:
```

Comment:
Put the subchart as the last section, this is what we normally do in other charts

Reply:
Moved [d027698](https://github.com/bitnami/charts/pull/13480/commits/d02769888284d04a9e4be0009261f54da4e9dbff)